### PR TITLE
Add directives in makefile

### PR DESCRIPTION
**Pull request by mmsc2**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,6 +32,11 @@ deps:
 	pip install cairo_lang==0.9.1 && \
 	deactivate
 
+maturin: 
+	python3 -m venv cairo-rs-py-env
+	cairo-rs-py-env/bin/activate
+	maturin develop
+
```
Comment (fmoletta) [inline Makefile:39]: Please remove unused target

File: Makefile [hunk 2]
```diff
@@ -1,12 +1,28 @@
-.PHONY: deps deps-macos deps-default-version build run check test clippy clean run-python-test  full-test
+.PHONY: deps deps-macos deps-default-version build run check test clippy clean run-python-test full-test
```
Comment (fmoletta) [inline Makefile:1]: Please add newly created target names here

File: Makefile [hunk 3]
```diff
@@ -59,4 +77,19 @@ run-python-test: $(COMPILED_TESTS)
...
 full-test: test run-python-test
+
+compare_trace_memory: $(CAIRO_RS_TRACE) $(CAIRO_TRACE) $(CAIRO_RS_MEM) $(CAIRO_MEM)
+	cd tests; ./compare_vm_state.sh trace memory
+
+compare_trace: $(CAIRO_RS_TRACE) $(CAIRO_TRACE)
+	cd tests; ./compare_vm_state.sh trace
+
+compare_memory: $(CAIRO_RS_MEM) $(CAIRO_MEM)
+	cd tests; ./compare_vm_state.sh memory
```
Comment (pefontana) [inline Makefile:95]: endline

**Review by fmoletta — APPROVED** [review]
LGTM

**Review by Oppen — APPROVED** [review]
LGTM

**Pull request merged**